### DISCUSSION TITLE
Welcome to localGoogoo Discussions!

### DISCUSSION THREAD

**Discussion by kodejuice** [kodejuice/localGoogle#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ajaxStardust** [comment]
Hi there. I was looking for a simple app to run on a local LAMP stack http server. (aka. web dev on localhost). 

I have your app setup, but it never crawls / indexes anything. 

I'm running Apache2, php 7.3, MySQL 8 on WSL2:Ubuntu (win 10). 

I've got directories with ./index.html and ./index.php

i've tried subdirectories, etc. Nothing seems to happen. Have you any recommendations? I did setup the included .json confg file w/ the correct db credentials. 

Thank you!

**Reply by kodejuice** [reply]
What url are you trying to crawl?

**Reply by kodejuice** [reply]
> Make sure the localgoogoo folder is placed somewhere under your local web document root. Your offline websites should also be under local web directory, localGoogoo wont be able to crawl them if they're not accessible via the http:// protocol.

**Reply by ajaxStardust** [reply]
Hi there! Thank you very much for the prompt reply!

I'm trying to index http://localhost/ 
As a web devr, I have a lot of files on a local HDD, my Apache2 DOCUMENT_ROOT is H:\Apache2\htdocs
Note that localGoogoo is a sub-dir of the DOCUMENT_ROOT, as shown:
![image](https://user-images.githubusercontent.com/94220040/185172599-e49840b3-062f-4bf7-815e-9c44690a93e5.png)

Here's a screen catpure of what i experience:
https://screenrec.com/share/UTNw28u06d

Maybe im expecting it to do something it's not designed to do: E.g. if you recall an app (which doesn't work anymore for PHP ver upgrades), "Sphider" did basically what I want to do: To type in a keyword, E.g. "[some client's business name]", and return a result showing where that doc is, in case I forget the actual container name of the website files. 

Best regards. And, BTW, I hope my rhetoric didn't imply that your app isn't any good. I'm confident that it is I who has misconfigured, or otherwise don't understand what's going on.

**Reply by kodejuice** [reply]
Oh, i see now
What does the `index.html` or `index.php` file in your htdocs folder contain?

It has to contain links that point to other pages, that's how the crawler works, it recursively visits the links in the page and indexes them.

To test that the crawling process actually works, you can try to crawl/index any online website.

**Reply by kodejuice** [reply]
You can create a `index.php` inside your `htdocs` folder that contains a link to all local website on your HDD, that way localgoogoo will crawl them all.

**Reply by kodejuice** [reply]
see this page:

![image](https://user-images.githubusercontent.com/9017620/185347159-70c46731-6503-41b9-bbc5-d53dfc413a5a.png)

For each site i want to crawl, they're under `htdocs`, so i use url `http://localhost/[site_or_folder_name]`, not just `http://localhost`
